### PR TITLE
Example bot

### LINKED ISSUE #3: Test bot to walk around and kill stuff
On start command by GameMaster

### PR DESCRIPTION
**Pull request by dariusf**
Fixes #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bot",
+  "version": "1.0.0",
+  "description": "",
+  "main": "bot.js",
+  "dependencies": {
+    "mineflayer": "1.7.5",
+    "mineflayer-blockfinder": "0.0.5",
```
Comment (LowWeiLin) [inline package.json]: Is this required? Can leave it here if it is not broken though.

**Comment by LowWeiLin** [comment]
I will try to add in mineflayer-auto-auth, so that it can register/login automatically on my server.

**Comment by LowWeiLin** [comment]
Updated

File: package.json (hunk 1 above)
Comment (dariusf) [reply package.json]: Not required, forgot to remove it.

**Review by kanghj — APPROVED** [review]

**Pull request merged**